website/content/docs/secrets/ssh/signed-ssh-certificates.mdx

```diff
@@ -466,6 +466,50 @@ forwarding. See [no prompt after login](#no-prompt-after-login) for examples.
 }
 ```
 
+### Key Comments
+There are additional steps needed to preserve [comment attributes](https://docs.oasis-open.org/kmip/kmip-spec/v2.1/cs01/kmip-spec-v2.1-cs01.html#_Toc32239331)
```

Comment:
I don't believe this is the correct link for this as KMIP is an entirely different spec. I think https://www.rfc-editor.org/rfc/rfc4253#section-6.6 is closer, but doesn't mention comments. https://www.rfc-editor.org/rfc/rfc4716#section-3.3.2 mentions comments, but uses the PEM-style key encoding rather than the wire-style key encoding. 

https://github.com/openssh/openssh-portable/blob/455cee8d6c2e4c48c5af9faead3599c49948411e/authfile.c#L261-L295 is the relevant code snippet but lacks docs as to format spec.

Either way, I'd remove the link to the KMIP spec.

Reply:
used the 2nd link - thank you @cipherboy